pallets/parachain-staking/src/lib.rs

```diff
@@ -1711,7 +1711,7 @@ pub mod pallet {
             // choose the top TotalSelected qualified candidates, ordered by stake
             let collators = Self::compute_top_candidates();
             if collators.is_empty() {
-                // SELECTION FAILED TO SELECT >=1 COLLATOR => select collators from previous round
+                log::error!("FAILED TO SELECT >=1 COLLATOR => using collators from previous round");
```

Comment:
Maybe you can add a tag to logging,
```
const LOG_TARGET: &str = "parachain-staking";
log::error!(target: LOG_TARGET, "FAILED TO SELECT >=1 COLLATOR => using collators from previous round");
```
then you can filter logs by the tag, like 
```
...
-lparachain-staking=error \
...
```

Reply:
i'm going to submit this change as a followup PR #1132 to not void the approvals